### ISSUE TITLE
Terminal.split_seqs issue with CUB/CUF

### ISSUE THREAD

**Issue by haliphax** [jquast/blessed#197]
When using `Terminal.split_seqs`, any CUB/CUF will take up two positions in the resulting list. The first is the sequence, and the second is the numerical argument for how far the cursor should move.

Reproduce:
```python
from blessed import Terminal
t = Terminal()
print(t.split_seqs(t.move_right(10)))
print(t.split_seqs(t.move_left(6)))
```

Outputs:
```
['\x1b[10C', '10']
['\x1b[6D', '6']
```

CUU (move_up) and CUD (move_down) do not exhibit this behavior.

**Comment by avylove** [comment]
The difference exists because  CUB and CUF have `match_grouped` set to `True` in the `CAPABILITY_DATABASE` while CUU and CUD don't. This is also set for CUP. I believe the reason is to support `Termcap.horizontal_distance()`

@jquast, I'm not sure the best place to address this.

**Comment by haliphax** [comment]
Yeah, I poked around through the Python side of things and didn't see any great way to get at it. I'm currently just handling the bug in my userland code by detecting when a CUB/CUF sequence is seen while looping through `split_seqs` output, and then skipping the next value in the list.

**Comment by avylove** [comment]
Fix is in [1.18.1](https://github.com/jquast/blessed/releases/tag/1.18.1). Let us know if you see any other issues.
